packages/rest/src/__tests__/acceptance/routing/routing.acceptance.ts

```diff
@@ -735,6 +735,61 @@ describe('Routing', () => {
         .get('/greet/world')
         .expect(200, 'HELLO WORLD');
     });
+
+    it('redirect swagger routes', async () => {
+      const app = new RestApplication();
+      await whenIMakeRequestTo(app)
+        .get('/swagger-ui')
+        .expect(308);
+      await whenIMakeRequestTo(app)
+        .get('/explorer')
+        .expect(308);
+    });
+
+    it('redirect controller routes', async () => {
+      class MyController {
+        @get('/hello', {
+          responses: {},
+        })
+        hello(): string {
+          return 'hello';
+        }
+        @get('/hello/world')
+        helloWorld() {
+          return `hello world`;
+        }
+      }
+      const app = givenAnApplication();
+      const server = await givenAServer(app);
+      server.redirect('/hello', '/hello/world');
+      givenControllerInApp(app, MyController);
+      const response = await whenIMakeRequestTo(server)
+        .get('/hello')
+        .expect(303);
+      await whenIMakeRequestTo(server)
+        .get(response.header.location)
+        .expect(200, 'hello world');
+    });
+
+    it('redirect routes based on their specifics', async () => {
```

Comment:
Similarly here. What scenario are you trying to verify in this test? It's not clear to me.

Reply:
https://github.com/strongloop/loopback-next/blob/78f70a1b49ac295b77336a1b5eb8f119abfd1815/packages/rest/src/__tests__/acceptance/routing/routing.acceptance.ts#L781

 make new request to the redirect route based on location  and verifying the result 
its more idealistic.